frontend/src/metabase/collections/components/CollectionSidebar/Bookmarks/Bookmarks.tsx

```diff
@@ -18,31 +19,22 @@ import { SidebarHeading } from "metabase/collections/components/CollectionSideba
 import { Bookmark, BookmarkableEntities, Bookmarks } from "metabase-types/api";
 
 interface LabelProps {
-  name: string;
-  type: BookmarkableEntities;
+  bookmark: Bookmark;
 }
 
 interface CollectionSidebarBookmarksProps {
   bookmarks: Bookmarks;
   deleteBookmark: (id: string, type: string) => void;
 }
 
-function getIconForEntityType(type: BookmarkableEntities) {
-  const icons = {
-    card: "grid",
-    collection: "folder",
-    dashboard: "dashboard",
-  };
-
-  return icons[type];
-}
+const Label = ({ bookmark }: LabelProps) => {
+  const iconName = getIcon(bookmark);
+  const isOfficial = bookmark.authority_level === "official";
```

Comment:
Please use `PLUGIN_COLLECTIONS` since official collections are a paid feature. [See an example at `CollectionBadge` component](https://github.com/metabase/metabase/blob/5fce46b3ff95959291d07b124420869620293341/frontend/src/metabase/questions/components/CollectionBadge.jsx#L29)